internal/cmd/asyncapi/command_import.go

```diff
@@ -0,0 +1,464 @@
+	OperationId string                        `yaml:"operationId"`
+	TopicTags   []spec2.Tag                   `yaml:"tags"`
+	OpBindings  spec2.OperationBindingsObject `yaml:"bindings"`
+	Message     MessageRef                    `yaml:"message"`
+}
+
+type Topic struct {
+	Description           string        `yaml:"description"`
+	Publish               Operation     `yaml:"publish"`
+	Subscribe             Operation     `yaml:"subscribe"`
+	XMessageCompatibility string        `yaml:"x-messageCompatibility"`
+	Bindings              TopicBindings `yaml:"bindings"`
+}
+
+type TopicBindings struct {
+	Kafka kafkaBinding `yaml:"kafka"`
+}
+
+func newImportCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import <specification-file>",
+		Short: "Populate cluster using AsyncAPI specification file.",
+		Long:  "Add topics, schemas, and tags from the AsyncAPI YAML specification file to the active Confluent Cloud cluster.",
+		Args:  cobra.ExactArgs(1),
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Import an AsyncAPI specification file to populate an existing Kafka cluster and Schema Registry.",
+				Code: "confluent asyncapi import spec.yaml",
+			},
+		),
+	}
+
+	c := &command{pcmd.NewAuthenticatedStateFlagCommand(cmd, prerunner)}
+	cmd.RunE = c.asyncapiImport
+
+	cmd.Flags().Bool("overwrite", false, "Overwrite existing topics with the same name.")
+	cmd.Flags().String("kafka-api-key", "", "Kafka cluster API key.")
+	cmd.Flags().String("schema-registry-api-key", "", "API key for Schema Registry.")
+	cmd.Flags().String("schema-registry-api-secret", "", "API secret for Schema Registry.")
+
+	return cmd
+}
+
+func getFlagsImport(cmd *cobra.Command) (*flagsImport, error) {
+	overwrite, err := cmd.Flags().GetBool("overwrite")
+	if err != nil {
+		return nil, err
+	}
+	kafkaApiKey, err := cmd.Flags().GetString("kafka-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiKey, err := cmd.Flags().GetString("schema-registry-api-key")
+	if err != nil {
+		return nil, err
+	}
+	schemaRegistryApiSecret, err := cmd.Flags().GetString("schema-registry-api-secret")
+	if err != nil {
+		return nil, err
+	}
+
+	flags := &flagsImport{
+		overwrite:               overwrite,
+		kafkaApiKey:             kafkaApiKey,
+		schemaRegistryApiKey:    schemaRegistryApiKey,
+		schemaRegistryApiSecret: schemaRegistryApiSecret,
+	}
+	return flags, nil
+}
+
+func (c *command) asyncapiImport(cmd *cobra.Command, args []string) error {
+	spec, err := fileToSpec(args[0])
+	if err != nil {
+		return err
+	}
+	// Get flags
+	flagsImp, err := getFlagsImport(cmd)
+	if err != nil {
+		return err
+	}
+	// Getting Kafka Cluster & SR
+	flagsExport := flags{
+		kafkaApiKey:             flagsImp.kafkaApiKey,
+		schemaRegistryApiKey:    flagsImp.schemaRegistryApiKey,
+		schemaRegistryApiSecret: flagsImp.schemaRegistryApiSecret,
+	}
+	details, err := c.getAccountDetails(&flagsExport)
+	if err != nil {
+		return err
+	}
```

Comment:
This is a better practice, since you only need to use `&` once if you pass the variable to multiple functions.
```suggestion
	flagsExport := &flags{
		kafkaApiKey:             flagsImp.kafkaApiKey,
		schemaRegistryApiKey:    flagsImp.schemaRegistryApiKey,
		schemaRegistryApiSecret: flagsImp.schemaRegistryApiSecret,
	}
	details, err := c.getAccountDetails(flagsExport)
	if err != nil {
		return err
	}
```